docs/ChainOrderTransitivity.md

```diff
@@ -0,0 +1,153 @@
+# Transitivity of the chain order
+
+## Context
+
+The abstract Consensus layer currently implements the chain order like this:
+From every header, one can extract a `SelectView`, which has an `Ord` instance.
+A chain `C` is _preferable_ to a chain `C'` if `selectView(C) > selectView(C')` where `C` and `C'` are the tip headers of `C` and `C'`, respectively.
+
+## Examples for intransitive orders
```

Comment:
It seems "intransitive" sometimes means something more specific. So I've been striving to write "not transitive", which is awkward :(

Reply:
https://github.com/IntersectMBO/ouroboros-consensus/pull/891#discussion_r1458775301